Umbraco-Cloud/Deployment/Hotfixes/index.md

```diff
@@ -0,0 +1,30 @@
+# How to handle hotfixes on Umbraco Cloud projects
+
+When you have an Umbraco Cloud project with multiple environments, you might run into a situation where you need to push a hotfix to your Live environment, but have other pending changes on your Development environment that is not yet ready for the Live site.
+
+Let's say you have 2 environments, a Live environment and a Development environment. You are currently working on building some changes on your local clone of the Development environment, but these changes won't be ready for the Live environment for another few weeks. Now you need to apply a minor change to your Live environment - a *hotfix*.
+
+Normally you would do this, by making the hotfix locally, push it to the Development environment and then deploy it to the Live environment. In this scenario that's not possible, as you do not want to deploy the other pending changes you are still working on.
+
+Following the workflow of Umbraco Cloud, you should never make changes directly on your Live environment. You should only make changes on the Live environment if that's the only environment you have. Read the [Project overview](../../Getting-Started/Project-overview/) article to learn more about environments on Umbraco Cloud.
+
+With that said, it **is** possible to apply selected changes to your Live environment without breaking the workflow of Umbraco Cloud. 
+
+There are two roads you can take:
+
+## [1. Move files manually](Move-files-manually/)
+
+One way to apply a hotfix to a Live environment would be to clone down both your Development and your Live environment to your local machine. Take the changed and/or updated files from the cloned Development environment and copy them into the cloned Live environment, and push them to the Live environment on Cloud. Using this guide will also allow you to test the changes on a cloned version of your Live environment before pushing to the Cloud.
+
+![Manual move](images/hotfix-manual-move.gif)
+
+Read the guide here: [Move files manually](Move-files-manually/)
+
+## [2. Use Git "magic"](Using-Git/)
```

Comment:
Not sure about the title here. It is not "magic" - its git 😉 
*Apply fix with Git branching* perhaps?

Reply:
Yeah, I don't really like the title either, but didn't really have any better suggestions 🙈 

I like _Apply fix with Git branching_ 🙏